topics/single-cell/tutorials/scrna-case_trajectories/tutorial.md

```diff
@@ -0,0 +1,444 @@
+> > 2. Cd4 and Cd8 expression appears highest in the DP-L cluster. The expression of both Cd4 and Cd8 also appears higher than we might expect in the DP-M4 cluster - perhaps this is a sign that it is closer to the DP-L cluster than it seems in this simple plot. 
+> >
+> > ![DN cluster is on one side of the plot, with the DP L and T mature clusters on the other side. The other clusters are close together in the centre of the plot. Coloured to show higher Cd4 and Cd8 expression in the M4, DP L and T mature clusters.](../../images/scrna-case_trajectories/TrajectoriesPAGA.png "PAGA plots coloured by cell type, Cd4 expression, and Cd8 expression")
+> {: .solution}
+>
+{: .question}
+
+## Re-draw force-directed graph (again!)
+
+Force directed graphs can be initialised randomly, or we can prod it in the right direction. We'll prod it with our PAGA calculations. Note that you could also try prodding it with tSNE or UMAP. A lot of these tools can be used on top of each other or with each other in different ways, this tutorial is just one example. Similarly, you could be using any **obs** information for grouping, so could do this for *louvain* or *cell_type* for instance.
+
+> <hands-on-title> Initialise FDG using PAGA </hands-on-title>
+>
+> 1. {% tool [Scanpy RunFDG](toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_run_fdg/scanpy_run_fdg/1.8.1+galaxy9) %} with the following parameters:
+>    - {% icon param-file %} *"Input object in AnnData/Loom format"*: `output_h5ad` (output of **Scanpy PlotTrajectory** {% icon tool %})
+>    - *"Use programme defaults"*: {% icon history-share %} `No`
+>    - *"Method to initialise embedding, any key for adata.obsm or choose from the preset methods"*: `paga`
+>    - *"Graph layout"*: `fa`
+>
+> 2. {% tool [Scanpy PlotEmbed](toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_plot_embed/scanpy_plot_embed/1.8.1+galaxy9) %} with the following parameters:
+>    - {% icon param-file %} *"Input object in AnnData/Loom format"*: `output_h5ad` (output of **Scanpy RunFDG** {% icon tool %})
+>    - *"name of the embedding to plot"*: `draw_graph_fa`
+>    - *"color by attributes, comma separated texts"*: `cell_type`
+>    - *"Use raw attributes if present"*: `No`
+>    - *"Location of legend"*: `On data`
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. How has basing our FDG on the PAGA plot changed the relationships between our cells?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. The interesting change here is that our DP-M4 cells are now heading on a clear trajectory towards differentiation. It looks like we've got the correct ordering of cells from DN through the DP groups and on towards T-mature. We didn't see this in our previous plots. 
+> >
+> > ![The M4 cluster is now closer to the DP L cluster](../../images/scrna-case_trajectories/TrajectoriesFDG3.png "FDG plot initiated from the PAGA plot")
```

Comment:
this is fantastic alt text, short and tells the reader exactly what they should be getting out of this graphic.